stereotype/meta.py

```diff
@@ -61,6 +64,13 @@ def _iterate_fields(mcs, field_annotations: Dict[str, Any]) -> Iterable[Tuple[st
         for name, annotation in field_annotations.items():
             if name.startswith('_'):
                 continue
+            # Skip attributes with ClassVar annotations - they shouldn't become fields
+            if isinstance(annotation, str):
+                # Delayed annotation; it's not a good time to resolve type hints properly, work with it as a string
+                if annotation.startswith('ClassVar[') and annotation.endswith(']'):
```

Comment:
This won't work in case the typehint is declared as `flag: "typing.ClassVar[bool] = False", i.e. if the whole `typing` module is imported. We obviously cannot do anything about stuff like renaming `ClassVar` to something else when it's imported, but I think we should handle at least this basic usecase.

```suggestion
                if (annotation.startswith('ClassVar[') or annotation.startswith('typing.ClassVar[')) and annotation.endswith(']'):
```

Reply:
Good point, change applied with some test coverage